docs/troubleshoots.md

```diff
@@ -0,0 +1,28 @@
+# TROUBLESHOOTS 
+
+## Setting up Gmail to be used as default mail server in .env
+
+Gmail required applications that will use their service to be authorized first beforehand. Failure in doing this will cause SMTP sending mail process fail with the response **"534-5.7.9 Application-specific password required..."**.
+
+To solve this issue, you need to give this application an App password that is generated from Gmail account.
+
+Here are the steps:
+
+1. Login to your Gmail account and click on **Security** tab from the top navbar.
+![signing in settings original](https://github.com/mtreacy002/mentorship-backend/blob/e4dbe9f00589fe5686904ed1276d5241132d5044/docs/images/signing_in_settings_gmail_original.png)
+
+2. In the section *Signing in to Google* select **2-step verification** and follow the prompts to turn it ON. 
+3. Once step 2 completed, go back to Security page and from the same section (*Signing in to Google*) select **App password**.
+![signing in settings with 2-steps on](https://github.com/mtreacy002/mentorship-backend/blob/e4dbe9f00589fe5686904ed1276d5241132d5044/docs/images/signing_in_settings_gmail_2steps_on.png)
+
+4. Under *Select the app and device you want to generate the app password for.*:
+          - for *Select app* choose **Mail**
+          - for *Select device* choose **Other (Custom name)** then type **MentorshipSystem** (or choose any name you like)
+![setting app passwords](https://github.com/mtreacy002/mentorship-backend/blob/e4dbe9f00589fe5686904ed1276d5241132d5044/docs/images/setting_gmail_app_passwords.png)
+
+5. Once you completed step 3 above, a window will open and show the autogenerated password that you can use for the application. Copy paste this code to your .env as the value for **APP_MAIL_PASSWORD**. Click done on the open window (back to Gmail account windor), then you will see it on the **App passwords** list.
+![new app password in list](https://github.com/mtreacy002/mentorship-backend/blob/e4dbe9f00589fe5686904ed1276d5241132d5044/docs/images/gmail_new_app_password_in_list.png)
+
+6. Make sure you have your **MAIL_SERVER** config in the .env set to **smtp.gmail.com** and both **MAIL_DEFAULT_SENDER** and  **APP_MAIL_USERNAME** set to the Gmail account you just authorized the app with.
+
+7. Save and run the python script again (*python run.py* from the command line) and try to register a new user on the swagger UI. This should give HTTP response 200 (successful).
```

Comment:
```suggestion
7. Save and run the python script again from the command line: `python run.py`. Then try to register a new user on the Swagger UI. This should give HTTP response 200 (successful).
```
"``" is markdown for in line code.

Reply:
@isabelcosta, I've updated the file as per your suggestions and have pushed the latest squash commit. Pls check and let me know what you think. Thanks again for the tips. 😃